packages/grafana-ui/src/components/Collapse/CollapsableSection.tsx

```diff
@@ -32,6 +32,10 @@ export const CollapsableSection: FC<Props> = ({
   const styles = useStyles2(collapsableSectionStyles);
   const tooltip = `Click to ${open ? 'collapse' : 'expand'}`;
   const onClick = (e: React.MouseEvent) => {
+    if (e.target instanceof HTMLElement && e.target.tagName === 'A') {
+      return;
+    }
```

Comment:
I'm not super enthusiastic about this. In general, I think we really should be moving way from nesting interactive elements within each other, because it just leads to all these sorts of weird corner cases that we have to keep carving out, which makes it hard to predict how a component will behave.

Side note - I noticed the onClick is on both the parent `<div>` and the `<button>`. Is that intentional?

Reply:
Yep, intentional. It's detailed [here](https://inclusive-components.design/cards/) under the "redundant click event" heading. The alternative approach here is use the pseudo-element trick to cover the card without nesting child elements, and then using `position: relative` to float any other interactive elements "above" the pseudo-element, but I remember you not being a fan of that approach either